### ISSUE TITLE
Typing comments always pops up the 'intellisense' drop-down

### ISSUE THREAD

**Issue by paweston70** [microsoft/azuredatastudio#19225]
Issue Type: <b>Bug</b>

*(I'm not sure if this is a bug or a feature change - I went off the limited choices of the 'Report Issue' dialog in ADS).*

Typing comments always pops up the 'intellisense' drop-down.  I always use a tab after a line comment (--\t), followed by the comment I want to write, e.g.

	--	My comment here.

What I'm getting, really annoyingly, is:

	--ASMy comment here.

...due to the fact that the 'AS' keyword is first in the intellisense list. Could we have it so that the intellinsense list is *not* available when we type a comment, please (either -- or /\*...\*/)?

Kind regards,
Paul.

Azure Data Studio version: azuredatastudio 1.35.1 (fab63efd307e54c063c8af03474bbb5d77f5fd11, 2022-03-15T18:03:38.261Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

**Comment by alanrenmsft** [comment]
@paweston70  I am not able to reproduce the issue, for me, as soon as I type the second `-` the suggestion list will go away
![comment](https://user-images.githubusercontent.com/13777222/166988409-8b2c4c9e-2112-4b52-b5ab-61a9fcb65a8f.gif)

could you please record your repro steps?

**Comment by alanrenmsft** [comment]
please reopen when you have the information that we requested.

**Comment by paweston70** [comment]
@alanrenmsft - Many apologies, I've been elsewhere doing other stuff since I posted this.

I'm not, presently, able to reproduce the issue. As soon as I can, I'll post a file to show the steps I took. One point of note is that this occurs when SQL is the language being used, but I have no SQL extensions installed.

There is a possibility that this could be coming from other parts of the same document.

Thanks for your time, Alan.
